stage/tests/lms/test_demo.py

```diff
@@ -0,0 +1,25 @@
+from bok_choy.web_app_test import WebAppTest
+from ...pages.lms.login_lms import LmsLogin
+from ...pages.lms.dashboard_lms import DashboardPageExtended
+
+
+class DemoTest(WebAppTest):
+    """
+    Sample Test for logging in and navigating to Courseware page
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
```

Comment:
I had a discussion with @Muddasser and it seems like he has already tackled this.